data/pathfinder/paizo/roleplaying_game/core_essentials/ce_abilities_race.lst

```diff
@@ -225,15 +225,16 @@ CATEGORY=Internal|Base Size ~ Colossal.MOD
-Pincers		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:PincersAttacks|0		BONUS:VAR|PincersAttacks|2	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Pincers|TOHIT|-5	BONUS:WEAPONPROF=Pincers|DAMAGE|-(STR/2)
+Pincers		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:PincersAttacks|0		BONUS:VAR|PincersAttacks|2	BONUS:VAR|NaturalAttacks|PincersAttacks			BONUS:WEAPONPROF=Pincers|TOHIT|-5	BONUS:WEAPONPROF=Pincers|DAMAGE|-(STR/2)
 Tail Slap		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:TailSlapAttacks|0	BONUS:VAR|TailSlapAttacks|1	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Tail Slap|TOHIT|-5	BONUS:WEAPONPROF=Tail Slap|DAMAGE|-(STR/2)
 Claw			CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:ClawAttacks|0		BONUS:VAR|ClawAttacks|2	BONUS:VAR|NaturalAttacks|ClawAttacks		BONUS:WEAPONPROF=Claw|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
 Slam			CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:SlamAttacks|0		BONUS:VAR|SlamAttacks|1	BONUS:VAR|NaturalAttacks|1				BONUS:WEAPONPROF=Slam|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
 Sting			CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:StingAttacks|0		BONUS:VAR|StingAttacks|1	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Sting|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
 Talons		CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:TalonsAttacks|0		BONUS:VAR|TalonsAttacks|2	BONUS:VAR|NaturalAttacks|TalonsAttacks	BONUS:WEAPONPROF=Talons|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
-Hoof			CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:HoofAttacks|0		BONUS:VAR|HoofAttacks|2	BONUS:VAR|NaturalAttacks|1				BONUS:WEAPONPROF=Hoof|TOHIT|-5	BONUS:WEAPONPROF=Hoof|DAMAGE|-(STR/2)
-Tentacle		CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:TentacleAttacks|0	BONUS:VAR|TentacleAttacks|2	BONUS:VAR|NaturalAttacks|1			BONUS:WEAPONPROF=Tentacle|TOHIT|-5	BONUS:WEAPONPROF=Tentacle|DAMAGE|-(STR/2)
-Wing			CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:WingAttacks|0		BONUS:VAR|WingAttacks|2	BONUS:VAR|NaturalAttacks|1				BONUS:WEAPONPROF=Wing|TOHIT|-5	BONUS:WEAPONPROF=Wing|DAMAGE|-(STR/2)
+Hoof			CATEGORY:Internal		TYPE:NaturalAttack.Secondary	DEFINE:NaturalAttacks|0	DEFINE:HoofAttacks|0		BONUS:VAR|HoofAttacks|2	BONUS:VAR|NaturalAttacks|HoofAttacks				BONUS:WEAPONPROF=Hoof|TOHIT|-5	BONUS:WEAPONPROF=Hoof|DAMAGE|-(STR/2)
+HoofPony	CATEGORY:Internal		TYPE:NaturalAttack.Primary	DEFINE:NaturalAttacks|0	DEFINE:HoofAttacks|0		BONUS:VAR|HoofAttacks|2	BONUS:VAR|NaturalAttacks|HoofAttacks				BONUS:WEAPONPROF=Hoof|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
```

Comment:
It’s not duplicate. The type is changed from TYPE.Secondary to TYPE.Primary. While the STR bonus and tohit bonus could be changed directly on the companion to “get the right number”, it would not actually change its hoof attack to primary, which it is. I don’t know if any other ability that depends on whether the attack is actually typed as primary or secondary. But just in case, that’s how I did it.

 

From: Andrew Maitland [mailto:notifications@github.com] 
Sent: Thursday, August 20, 2015 7:19 PM
To: PCGen/pcgen <pcgen@noreply.github.com>
Cc: mjmeans <mjmeans@ultrasw.com>
Subject: Re: [pcgen] More companion work (#477)

 

In data/pathfinder/paizo/roleplaying_game/core_essentials/ce_abilities_race.lst <https://github.com/PCGen/pcgen/pull/477#discussion_r37601265> :

>  Tail Slap           CATEGORY:Internal              TYPE:NaturalAttack.Secondary        DEFINE:NaturalAttacks|0        DEFINE:TailSlapAttacks|0        BONUS:VAR|TailSlapAttacks|1    BONUS:VAR|NaturalAttacks|1                    BONUS:WEAPONPROF=Tail Slap|TOHIT|-5   BONUS:WEAPONPROF=Tail Slap|DAMAGE|-(STR/2)
>  Claw                CATEGORY:Internal              TYPE:NaturalAttack.Primary        DEFINE:NaturalAttacks|0        DEFINE:ClawAttacks|0           BONUS:VAR|ClawAttacks|2        BONUS:VAR|NaturalAttacks|ClawAttacks          BONUS:WEAPONPROF=Claw|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
>  Slam                CATEGORY:Internal              TYPE:NaturalAttack.Primary        DEFINE:NaturalAttacks|0        DEFINE:SlamAttacks|0           BONUS:VAR|SlamAttacks|1        BONUS:VAR|NaturalAttacks|1                    BONUS:WEAPONPROF=Slam|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
>  Sting                  CATEGORY:Internal              TYPE:NaturalAttack.Primary        DEFINE:NaturalAttacks|0        DEFINE:StingAttacks|0          BONUS:VAR|StingAttacks|1       BONUS:VAR|NaturalAttacks|1                    BONUS:WEAPONPROF=Sting|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
>  Talons              CATEGORY:Internal              TYPE:NaturalAttack.Primary        DEFINE:NaturalAttacks|0        DEFINE:TalonsAttacks|0         BONUS:VAR|TalonsAttacks|2      BONUS:VAR|NaturalAttacks|TalonsAttacks        BONUS:WEAPONPROF=Talons|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1
> -Hoof                CATEGORY:Internal              TYPE:NaturalAttack.Secondary        DEFINE:NaturalAttacks|0        DEFINE:HoofAttacks|0           BONUS:VAR|HoofAttacks|2        BONUS:VAR|NaturalAttacks|1                    BONUS:WEAPONPROF=Hoof|TOHIT|-5       BONUS:WEAPONPROF=Hoof|DAMAGE|-(STR/2)
> -Tentacle            CATEGORY:Internal              TYPE:NaturalAttack.Secondary        DEFINE:NaturalAttacks|0        DEFINE:TentacleAttacks|0        BONUS:VAR|TentacleAttacks|2    BONUS:VAR|NaturalAttacks|1                    BONUS:WEAPONPROF=Tentacle|TOHIT|-5    BONUS:WEAPONPROF=Tentacle|DAMAGE|-(STR/2)
> -Wing                CATEGORY:Internal              TYPE:NaturalAttack.Secondary        DEFINE:NaturalAttacks|0        DEFINE:WingAttacks|0           BONUS:VAR|WingAttacks|2        BONUS:VAR|NaturalAttacks|1                    BONUS:WEAPONPROF=Wing|TOHIT|-5       BONUS:WEAPONPROF=Wing|DAMAGE|-(STR/2)
> +Hoof                CATEGORY:Internal              TYPE:NaturalAttack.Secondary        DEFINE:NaturalAttacks|0        DEFINE:HoofAttacks|0           BONUS:VAR|HoofAttacks|2        BONUS:VAR|NaturalAttacks|HoofAttacks                  BONUS:WEAPONPROF=Hoof|TOHIT|-5       BONUS:WEAPONPROF=Hoof|DAMAGE|-(STR/2)
> +HoofPony    CATEGORY:Internal              TYPE:NaturalAttack.Primary        DEFINE:NaturalAttacks|0        DEFINE:HoofAttacks|0           BONUS:VAR|HoofAttacks|2        BONUS:VAR|NaturalAttacks|HoofAttacks                  BONUS:WEAPONPROF=Hoof|DAMAGE|MAX(STR/2,0)|PREVAREQ:NaturalAttacks,1

Is there a reason we needed to add a duplicate attack when the fix could be placed directly on the Companion?

—
Reply to this email directly or view it on GitHub <https://github.com/PCGen/pcgen/pull/477/files#r37601265> .  <https://github.com/notifications/beacon/ADyapyL2pq3XUKe__2VXM1j2wSky4EeAks5opoIlgaJpZM4Fvktq.gif>